cineast-api/src/main/java/org/vitrivr/cineast/api/messages/query/TemporalQueryV2.java

```diff
@@ -0,0 +1,84 @@
+package org.vitrivr.cineast.api.messages.query;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
+import org.vitrivr.cineast.api.messages.interfaces.MessageType;
+import org.vitrivr.cineast.core.config.QueryConfig;
+import org.vitrivr.cineast.core.config.ReadableQueryConfig;
+
+/**
+ * This object represents a temporal-query message of temporal query version 2, i.e. a request for a
+ * temporally staged similarity-search.
+ */
+public class TemporalQueryV2 extends Query {
```

Comment:
I’m not opposed to deleting the old logic, our thinking was that it would be nice for backwards compatibility. But i think besides vitrivr-ng no other system uses the TemporalQuery Websocket API, so @vGsteiger you can remove the old logic.

Reply:
I don't think anything is using the websocket apart from vitrivr-ng, since the protocol handling needs to be done manually.

Reply:
Resolved in my last commits